tests/target/issue-5066/multi_line_struct_with_trailing_comma_always.rs

```diff
@@ -0,0 +1,8 @@
+// rustfmt-trailing_comma: Always
+// rustfmt-struct_lit_single_line: false
+
+fn main() {
+    let Foo {
+        a, ..
```

Comment:
I'm not sure if this is related to this issue, but this looks wrong to me. Seems like on a vertical layout like this (regardless of whether that happens due to width or other config options) there should still be a line break between each field

Reply:
Just to be clear you'd expect this to be the output?

```rust
let Foo {
    a,
    ..
} = b;
```

I don't think it's directly related to the duplicate comma, but I'm happy to make the change if you'd like.

Reply:
> Just to be clear you'd expect this to be the output?

Correct :+1: 

> I don't think it's directly related to the duplicate comma, but I'm happy to make the change if you'd like.

Fair enough, and def. don't want to add scope creep. Would you mind just confirming that it's reproducible behavior today (I suspect tuning down `struct_lit_width` to an absurdly low value might be a way to reproduce)?

If so then we can defer this to future discussion/investigation